CHANGELOG.rst

```diff
@@ -3,7 +3,7 @@ Changelog
 
 master
 ------
-
+- (`#69 <https://github.com/znicholls/silicone/pull/69>`_) Fixed bug that allowed RMS closest to select unfilled data
```

Comment:
```suggestion
- (`#69 <https://github.com/znicholls/silicone/pull/69>`_) Fixed bug so that ``DatabaseCruncherRMSClosest`` no longer selects scenarios which don't have follower data
```